### PR TITLE
update commands to cater for new attributes

### PR DESCRIPTION
**Pull request by DannyDakota**
update commands to cater for new attributes(faculty, matriculationNumber and covidStatus) and update tests, test data files to allow for PR#33 to pass CI

### CODE REVIEW

**Review by cwq2326 — APPROVED** [review]
All changes to the test cases conform to the new fields and updated GUI. 👍  LGTM 😄

File: src/test/java/seedu/address/logic/parser/AddCommandParserTest.java [hunk 1]
```diff
@@ -43,99 +52,141 @@
 public class AddCommandParserTest {
     private AddCommandParser parser = new AddCommandParser();
 
+    /**
```
Comment (jiale-c) [inline src/test/java/seedu/address/logic/parser/AddCommandParserTest.java:55]: I think is great that you added the javadoc and comments for this test cases to indicate what we are testing! :+1:

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 2]
```diff
@@ -143,17 +157,21 @@ public EditPersonDescriptor() {}
          */
         public EditPersonDescriptor(EditPersonDescriptor toCopy) {
             setName(toCopy.name);
+            setFaculty(toCopy.faculty);
             setPhone(toCopy.phone);
             setEmail(toCopy.email);
             setAddress(toCopy.address);
+            setMatriculationNumber(toCopy.matriculationNumber);
```
Comment (jiale-c) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:164]: Great use of SLAP here to make the code look organised! :muscle:

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 3]
```diff
@@ -188,6 +214,22 @@ public void setAddress(Address address) {
             return Optional.ofNullable(address);
         }
 
+        public void setMatriculationNumber(MatriculationNumber matriculationNumber) {
```
Comment (jiale-c) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:217]: The method naming is good. Shows Verbs for Actions here. :grin:

**Review by jiale-c — COMMENTED** [review]
LGTM too! The changes adhere to standard coding style and fit to our scope of a covid tracker application.

**Pull request merged**